assets/xml/objects/object_ik.xml

```diff
@@ -1,36 +1,46 @@
 ﻿<Root>
     <File Name="object_ik" Segment="6">
```

Comment:
Can you also add a comment explaining what this object is for between `Root` and `File`?